tests/docs-screenshots.spec.ts

```diff
@@ -38,17 +39,35 @@ test("instances", async ({ page }) => {
     .getByRole("button")
     .click();
   await page.getByText("Disk").click();
+  await page.waitForTimeout(1000);
   await page.screenshot({
     path: "tests/screenshots/doc/images/instances/create_instance_form_disk_devices.png",
     clip: getClipPosition(240, 0, 1280, 675),
   });
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_attach_to_instance_1.png",
+    clip: getClipPosition(240, 0, 1280, 675),
+  });
 
   await page.getByRole("button", { name: "Create override" }).click();
   await page.screenshot({
     path: "tests/screenshots/doc/images/instances/create_instance_form.png",
     clip: getClipPosition(240, 0, 1280, 675),
   });
 
+  await page.getByRole("button", { name: "Attach disk device" }).click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_attach_to_instance_2.png",
+    clip: getClipPosition(310, 230, 900, 480),
+  });
+
+  await page.getByRole("button", { name: "Attach custom volume" }).click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_attach_to_instance_3.png",
```

Comment:
This needs to ensure the content is loaded

<img width="1920" height="972" alt="image" src="https://github.com/user-attachments/assets/c1f0cce6-367e-4656-907e-5a27e0c316ed" />